lib/manager/dockerfile/extract.spec.ts

```diff
@@ -161,4 +170,41 @@ describe(getName(), () => {
       expect(getDep(null)).toMatchSnapshot();
     });
   });
+  describe('splitImageParts()', () => {
+    it('handles docker hub', () => {
+      expect(splitImageParts('node:14')).toMatchInlineSnapshot(`
+        Object {
+          "currentDigest": undefined,
+          "currentValue": "14",
+          "depName": "node",
+        }
+      `);
+    });
+    it('handles docker hub with full host', () => {
+      expect(splitImageParts('docker.io/node:14')).toMatchInlineSnapshot(`
+        Object {
+          "currentDigest": undefined,
+          "currentValue": "14",
+          "depName": "node",
+          "registryUrls": Array [
+            "https://docker.io",
+          ],
+        }
+      `);
+    });
+
+    it('handles gitlab project images', () => {
+      expect(splitImageParts('registry.mygitlab.test/jobs/docker/base:latest'))
+        .toMatchInlineSnapshot(`
+        Object {
+          "currentDigest": undefined,
+          "currentValue": "latest",
+          "depName": "docker/base",
+          "registryUrls": Array [
+            "https://registry.mygitlab.test/jobs",
```

Comment:
Did you validate that docker datasource will build the correct final url?

The `/v2/` path needs to be before `/jobs/`!

`https://registry.mygitlab.test/v2/jobs/docker/base/...`

Reply:
Yes there's a new function to ensure path prefix